src/api/manager/job/syncInventory.ts

```diff
@@ -37,7 +37,8 @@ async function getSourcesFromAPI() {
           audio_stream: {
             number_of_channels: source?.audio_stream?.number_of_channels,
             sample_rate: source?.audio_stream?.sample_rate
-          }
+          },
+          createdAt: new Date()
```

Comment:
I think this would conflict with the `lastConnected` field ( same purpose ish ) that we are adding in another ticket.

Reply:
Waiting on merge from PR https://github.com/Eyevinn/orchestration-gui/pull/3 to fix this comment.